src/deploy/functions/errorHandler.ts

```diff
@@ -34,11 +34,31 @@ export class ErrorHandler {
     if (this.errors.length === 0) {
       return;
     }
-    logger.info("\nFunctions deploy had errors with the following functions:");
+    logger.info("");
+    logger.info("Functions deploy had errors with the following functions:");
     for (const failedDeployment of this.errors) {
       logger.info(`\t${getFunctionLabel(failedDeployment.functionName)}`);
     }
-    logger.info("\nTo try redeploying those functions, run:");
+
+    const failedIamCalls = this.errors.filter((e) => e.operationType === "set invoker");
```

Comment:
changed this to error type since we want the function deployment to return a non-zero exit code